packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <form
     v-if="props.gql.browsers"
-    @submit.prevent="$emit('launch')"
+    @submit.prevent="emit('launch', props.gql.selectedBrowser.id)"
```

Comment:
What changed that we need to now include the id? I don't see that change anywhere in this PR.

Reply:
Now when we launch we are storing their selected browser, now that I think about it I might be able to pull this off of the graphql context since we store the selected browser. I needed to refactor this anyways because id seems to come from graphql but not actually on our `FoundBrowser` type